src/error/ColumnMetadataNotFound.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ *
+ */
+export class ColumnMetadataNotFoundError extends Error {
+    name = "ColumnMetadataNotFoundError";
+
+    constructor(propertyPath: string) {
+        super();
+        Object.setPrototypeOf(this, ColumnMetadataNotFoundError.prototype);
+        this.message = `No column metadata for property "${propertyPath}" was found.`;
```

Comment:
probably we need to give more user-friendly error. Users don't know what "metadata" means.

Reply:
You are right. We need more user friendly errors. What do you suggest?
EntityColumnNotFound -> `No entity column "${propertyPath}" was found.`?